### DISCUSSION TITLE
Welcome to bootstrap-email Discussions!

### DISCUSSION THREAD

**Discussion by stuyam** [bootstrap-email/bootstrap-email#77] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. How do you use Bootstrap Email?

**Comment by rprosenc** [comment]
Hi. 
I just stumbeld about this project, and it seems very interesting. What I need, is some kind of processing tool for HTML code generated by a PHP application. The documentation shows a usecase enclosed completely within a ruby application. As I know nothing about ruby, I initially imagined the following workflow as a possible integration scenario: a ruby app, that can be triggered via  command line (or http if not possible otherwise), which takes a path to some html file as argument, and returns some email-ready html code back. 
Is this, in principle, possible? 
(I saw requirements for fixed filenames and such, like in action_mailer_layout, this would be a problem)

Thank you very much and kind regards.

**Reply by stuyam** [reply]
For now I would suggest using https://editor.bootstrapemail.com to compile emails. However a new major version of the library is under development and it will support a command line feature 👍
